modules/standard/List.chpl

```diff
@@ -105,12 +105,12 @@ module List {
   pragma "no doc"
   proc _checkType(type eltType) {
     if isGenericType(eltType) {
-      compilerWarning("creating a list with element type " +
+      compilerWarning("creating a heap with element type " +
```

Comment:
did we make the wrong change here?

Reply:
(This is the List code, but this change should have happened in the Heap module)